pkg/sysregistriesv2/system_registries_v2_test.go

```diff
@@ -675,6 +675,72 @@ func TestPullSourcesFromReference(t *testing.T) {
 	assert.Equal(t, 1, len(pullSources))
 }
 
+func TestPullSourcesMirrorFromReference(t *testing.T) {
+	sys := &types.SystemContext{
+		SystemRegistriesConfPath:    "testdata/pull-sources-mirror-reference.conf",
+		SystemRegistriesConfDirPath: "testdata/this-does-not-exist",
+	}
+	registries, err := GetRegistries(sys)
+	assert.Nil(t, err)
+	assert.Equal(t, 4, len(registries))
+
+	for _, tc := range []struct {
+		registry      string
+		digestRef     string
+		digestSources int
+		tagRef        string
+		tagSources    int
```

Comment:
`digestSources, tagSources []string`, listing the exact expected values, would be nice.